app/webpack/src/dropzone.js

```diff
@@ -24,22 +24,24 @@ const ACCEPTED_FILES = [
 function addErrorMessage (msg) {
   // this adds an error message to the gov uk error summary and shows the errors
   const errorSummary = document.querySelector('.govuk-error-summary')
-  const li = errorSummary.querySelector('li')
-  let a = li.querySelector('a')
-  if (a === null) {
-    a = document.createElement('a')
-    li.appendChild(a)
-  }
+  const ul = errorSummary.querySelector('ul');
+  let li = document.createElement('li');
+  let a = document.createElement('a');
+  li.appendChild(a);
+  ul.appendChild(li);
   // add text and link to field
-  a.innerText = msg
+  a.innerText += msg
   a.setAttribute('aria-label', msg)
   a.setAttribute('data-turbolinks', false)
   a.setAttribute('href', '#dz-upload-button')
   // show error message on the dropzone form field
   const dropzoneElem = document.querySelector('#dropzone-form-group')
   dropzoneElem.classList.add('govuk-form-group--error')
   const fieldErrorMsg = document.querySelector('#dropzone-error')
-  fieldErrorMsg.innerText = msg
+  let div = document.createElement('div');
+  div.innerText = msg;
+  fieldErrorMsg.appendChild(div);
+  // fieldErrorMsg.innerText += msg
```

Comment:
Can this be deleted?

Reply:
Yes it can, good spot! :D
I carefully checked for rogue `console.logs` 🤦‍♂️